docs/gen-ai/llm-spans.md

```diff
@@ -36,49 +50,356 @@ Instrumentations that support it, MUST offer the ability to turn off capture of
+
+<!-- semconv gen_ai.user.message(full) -->
+The event name MUST be `gen_ai.user.message`.
 
 | Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
 |---|---|---|---|---|---|
-| [`gen_ai.completion`](../attributes-registry/llm.md) | string | The full response received from the LLM. [1] | `[{'role': 'assistant', 'content': 'The capital of France is Paris.'}]` | `Conditionally Required` if and only if corresponding event is enabled | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| [`gen_ai.system`](../attributes-registry/llm.md) | string | The name of the LLM foundation model vendor. | `openai` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 
-**[1]:** It's RECOMMENDED to format completions as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
+`gen_ai.system` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `openai` | OpenAI | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
 <!-- endsemconv -->
 
-[DocumentStatus]: https://github.com/open-telemetry/opentelemetry-specification/tree/v1.22.0/specification/document-status.md
+| Body Field | Type | Description | Examples | Requirement Level | Sensitive |
+|---|---|---|---|---|---|
+| `role` | string | The role of the messages author | `user` | `Opt-In` |  |
+| `content` | `AnyValue` | The contents of the user message. | `What telemetry is reported by OpenAI instrumentations?` | `Opt-In` | ![Sensitive](https://img.shields.io/badge/-sensitive-red) |
+
+### Assistant message
+
+This event describes the assistant message.
+
+<!-- semconv gen_ai.assistant.message(full) -->
+The event name MUST be `gen_ai.assistant.message`.
+
+| Attribute  | Type | Description  | Examples  | [Requirement Level](https://opentelemetry.io/docs/specs/semconv/general/attribute-requirement-level/) | Stability |
+|---|---|---|---|---|---|
+| [`gen_ai.system`](../attributes-registry/llm.md) | string | The name of the LLM foundation model vendor. | `openai` | `Recommended` | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+`gen_ai.system` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
+
+| Value  | Description | Stability |
+|---|---|---|
+| `openai` | OpenAI | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+<!-- endsemconv -->
+
+| Body Field | Type | Description | Examples | Requirement Level | Sensitive |
+|---|---|---|---|---|---|
+| `role` | string | The role of the messages author | `assistant` | `Conditionally Required`: when the message is returned by GenAI model. |  |
```

Comment:
This says `Conditionally Required` while the `role` in `gen_ai.user.message` says `Opt-in`. Are they different?

Reply:
I removed role for all events. We can always add it back later if it's needed. For now it seems to duplicate the event name.